src/app/modules/spcp/spcp.controller.ts

```diff
@@ -184,3 +186,112 @@ export const handleLogin: (
       return res.redirect(destination)
     })
 }
+
+/**
+ * Handler to return the public JWKS hosted on the app's well known end point
+ * @returns public JWKS in json format
+ */
+export const handleGetWellKnown: ControllerHandler<
+  unknown,
+  Record<string, unknown>,
+  unknown,
+  unknown
+> = async (_req, res) => {
+  return res.json(SpOidcService.getRpPublicJwks())
```

Comment:
/nit: json-encoding at every request feels a little wasted since the json data is actually static and extracted from a json blob. Perhaps we could return the file from `spOidcRpJwksPublicPath` instead.

Reply:
Possible too. Right now the approach is to load the public key into the client on instantiation, and then retrieve it from there for the jwks endpoint - this ensures that it is always in sync. But ok to return the file directly too in the controller